xml/System/DateTime.xml

```diff
@@ -2033,7 +2033,7 @@ For applications in which portability of date and time data or a limited degree
 
  The fractional part of `value` is the fractional part of a day. For example, 4.5 is equivalent to 4 days, 12 hours, 0 minutes, 0 seconds, 0 milliseconds, and 0 ticks.
 
- The `value` parameter is rounded to the nearest millisecond.
+ The `value` parameter is no longer rounded to the nearest millisecond (as it used to be before .NET 7.0).
```

Comment:
Can we use what @tannergooding suggested in the comment https://github.com/dotnet/dotnet-api-docs/issues/8292#issuecomment-1211390405?

`The value parameter is rounded to at least the nearest millisecond. More accuracy may be available for certain inputs provided they represent less than 2^53 ticks.`

Reply:
I'm not sure that this comment makes it clearer the the documentation reader. There is no rounding of the `value` parameter in the code anymore. I think that @tannergooding's comment applies to the precision of the value, but can be misanderstood as if it implied that some rounding was performed.

Reply:
What about using something like:

 `The Add result is rounded to at least the nearest millisecond. In .NET 7.0 and up more accuracy may be available for certain inputs provided they represent less than 2^53 ticks.`

Reply:
To be clear, I want the doc at least to give some hints on how the result is calculated to set the expectation.

Reply:
How about `In .NET 7 and up, the double precision of the value parameter is used to round to the nearest tick`?

Reply:
Looks good to me.

Reply:
"In .NET 7 and later versions..."

Reply:
> I think that @tannergooding's comment applies to the precision of the value, but can be misanderstood as if it implied that some rounding was performed.

You must always assume implicit rounding may exist when dealing with `float`/`double`, even if no explicit rounding happens in code.

Simply doing `double x = someInt64;` is a potentially lossy cast and its one of the few places where `implicit casts` can result in a "loss of data". The same goes for `float x = someInt32;` (`int32` and `uint32` are lossless for `double`; as are `int16` and `uint16` for `float`; and `byte`/`sbyte` for `Half`).